ddtrace/constants.py

```diff
@@ -58,3 +58,29 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+APPSEC_BLOCKED_RESPONSE_HTML = """
+<!DOCTYPE html><html lang="en"><head> <meta charset="UTF-8"> <meta name="viewport"
+content="width=device-width,initial-scale=1"> <title>You've been blocked</title>
+<style>a, body, div, html, span{margin: 0; padding: 0; border: 0; font-size: 100%;
+font: inherit; vertical-align: baseline}body{background: -webkit-radial-gradient(26% 19%,
+circle, #fff, #f4f7f9); background: radial-gradient(circle at 26% 19%, #fff, #f4f7f9); display:
+-webkit-box; display: -ms-flexbox; display: flex; -webkit-box-pack: center; -ms-flex-pack: center;
+justify-content: center; -webkit-box-align: center; -ms-flex-align: center; align-items: center;
+-ms-flex-line-pack: center; align-content: center; width: 100%; min-height: 100vh; line-height: 1;
+flex-direction: column}p{display: block}main{text-align: center; flex: 1; display: -webkit-box;
+display: -ms-flexbox; display: flex; -webkit-box-pack: center; -ms-flex-pack: center;
+justify-content: center; -webkit-box-align: center; -ms-flex-align: center; align-items: center;
+-ms-flex-line-pack: center; align-content: center; flex-direction: column}p{font-size: 18px;
+line-height: normal; color: #646464; font-family: sans-serif; font-weight: 400}a{color:
+#4842b7}footer{width: 100%; text-align: center}footer p{font-size: 16px}</style></head><body>
+<main> <p>Sorry, you cannot access this page. Please contact the customer service team.</p></main>
+<footer> <p>Security provided by
+<a href="https://www.datadoghq.com/product/security-platform/application-security-monitoring/"
+target="_blank">Datadog</a></p></footer></body></html>"""
+
+# noqa: E501
```

Comment:
what line is this meant to be associated with?

Reply:
The line above before it was reformatted, removed on next commit.